core/pom.xml

```diff
@@ -135,6 +135,12 @@
       <version>${asciidoctorj-pdf.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <version>1.15</version>
+    </dependency>
+
 
     <dependency>
       <groupId>org.jboss.arquillian.junit</groupId>
```

Comment:
I also had classpath issues with snakeyaml  but it was caused by duplicate entries (e.g selenium-server also uses it)  but that would happen with others libs as well. I like using yaml to config because it looks pretty clean e.g. filling a map of (dynamic) properties is one line of code : yaml.load(attrs). Also other arquillian extensions are using it (chameleon, cube and persistence) so maybe we should ask them if they had conflicts. 

About using several entries will also work but not the best approach in my opinion.